### PR TITLE
Move the release section from learn.eml page to home.eml file

### PR DESCRIPTION
**Pull request by JanaganSaravanan**
Release section moved to home page from learn page.

### CODE REVIEW

**Comment by sabine** [comment]
Thanks @JanaganSaravanan!

There's also some HTML UI associated with the releases which would also need to move to `home.eml`. :slightly_smiling_face:

**Comment by JanaganSaravanan** [comment]
Hi, I have added the html ui associated with release section. please check the changes.

**Comment by JanaganSaravanan** [comment]
And please let me know if there is any changes i can correct.

File: src/ocamlorg_frontend/pages/home.eml [hunk 1]
```diff
@@ -1,10 +1,12 @@
 let render () =
 Layout.render
+~(release : Data.Release.t)
 ~use_swiper:true
 ~title:"Welcome to a World of OCaml"
 ~description:"OCaml is a general-purpose, industrial-strength programming language with an emphasis on expressiveness and safety."
 ~canonical:""
 ~banner:true @@
+~(release : Data.Release.t)
```
Comment (sabine) [inline src/ocamlorg_frontend/pages/home.eml]: I think `release` needs to be a parameter of the `render` function in line 1, instead.

**Review by sabine — COMMENTED** [review]
Cool. So it would be really useful if the code compiles.

If you're on MacOS or some Unix-derivative, this should work:

1. Have OCaml installed (see https://ocaml.org/install),
2. run
```
make switch

make start
```

when you're in your ocaml.org folder.

On Windows, we don't test ocaml.org, so there may be problems (or maybe not) because we don't run automated tests for building ocaml.org on Windows.

File: src/ocamlorg_frontend/pages/home.eml (hunk 1 above)
Comment (JanaganSaravanan) [reply src/ocamlorg_frontend/pages/home.eml]: I use iOS. i am afraid i can’t compile the code. Anyway if there is anything i can do please let me know.

Comment (sabine) [reply src/ocamlorg_frontend/pages/home.eml]: Oh I see, yes, it is very hard to do this from a mobile device, unfortunately.

Comment (JanaganSaravanan) [reply src/ocamlorg_frontend/pages/home.eml]: Sorry for the inconvenience. I will try to compile code. If I need anything I will get back to you.

Comment (JanaganSaravanan) [reply src/ocamlorg_frontend/pages/home.eml]: i couldn’t test it on macOS or Linux. what else can i do.please let me know.

**Review by sabine — APPROVED** [review]
I fixed it up. As a note for future contributions: It is necessary to be able to build and run the project to make truly helpful contributions that take work off of our shoulders. :slightly_smiling_face:

**Comment by JanaganSaravanan** [comment]
Hi! I apologize for the inconvenience. From next time I will try to build and run the projects. I really appreciate your feedback.

**Pull request merged**